Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,319 @@
+import demistomock as demisto
+from CommonServerPython import *
+from cbc_sdk import endpoint_standard
+from cbc_sdk import CBCloudAPI
+from cbc_sdk import errors
+import ntpath
+
+''' IMPORTS '''
+
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+CBD_LR_PREFIX = 'cbd-lr'
+
+
+# Using Py API
+def put_file_command(credentials: Dict, sensor_id: int, destination_path: str, file_id: str):
+    api = CBCloudAPI(**credentials)
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    file_path = demisto.getFilePath(file_id).get('path')
+    session.put_file(open(file_path, 'rb'), destination_path)
+    return f'File: {file_id} is now exist in the remote destination {destination_path}'
+
+
+# Using Py API
+def get_file_command(credentials: Dict, sensor_id: int, source_path: str, timeout: int = None,
+                     delay: float = None):
+    api = CBCloudAPI(**credentials)
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    if timeout:
+        timeout = int(timeout)
+    if delay:
+        delay = float(delay)
+    file_data = session.get_file(file_name=source_path, timeout=timeout, delay=delay)
+    file_name = ntpath.split(source_path)[1]
+    return fileResult(file_name, file_data)
+
+
+# Using Py API
+def delete_file_command(credentials: Dict, sensor_id: int, source_path: str):
+    api = CBCloudAPI(**credentials)
+
+    session = api.select(endpoint_standard.Device, sensor_id).lr_session()
+    session.delete_file(filename=source_path)
+    return f'The file: {source_path} was deleted'
```

Comment:
```suggestion
    return f'The file: {source_path} was deleted successfully.'
```

Add the "successfully" part to all of the success. messages.